### PR TITLE
Add fullscreen option in app setting

### PR DESCRIPTION
**Pull request by neopilou**
Added an option in the app configuration to switch in full screen

### CODE REVIEW

**Comment by dshokouhi** [comment]
Can you post a screenshot of the new setting?

**Comment by neopilou** [comment]
Screenshot_20200117-081213_Home Assistant


Video : https://www.dropbox.com/s/uz8t10n7nhma5t9/20200117_082216.mp4?dl=0

File: app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt [hunk 1]
```diff
@@ -226,6 +226,29 @@ class WebViewActivity : AppCompatActivity(), io.homeassistant.companion.android.
         presenter.onViewReady()
     }
 
+    override fun onWindowFocusChanged(hasFocus: Boolean) {
```
Comment (JBassett) [inline app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt]: I'm not sure we need to do this here, we should be able to just set full screen in the onResume.

File: app/src/main/java/io/homeassistant/companion/android/webview/WebViewPresenterImpl.kt [hunk 2]
```diff
@@ -70,6 +73,15 @@ class WebViewPresenterImpl @Inject constructor(
         }
     }
 
+    override fun isFS(key: String): Boolean {
```
Comment (JBassett) [inline app/src/main/java/io/homeassistant/companion/android/webview/WebViewPresenterImpl.kt]: Not sure why this is taking a parameter?  Can we just rename to `isFullScreen()`?

Comment (neopilou) [reply app/src/main/java/io/homeassistant/companion/android/webview/WebViewPresenterImpl.kt]: isFS () function parameter has been removed

File: app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt (hunk 1 above)
Comment (neopilou) [reply app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt]: Fullscreen is now set in onResume()

**Comment by neopilou** [comment]
Conflicts have been resolved in all files, but I cannot mark them as resolved ...

**Comment by barrymossel** [comment]
How can I test?

**Comment by JBassett** [comment]
It might be easier if you take your change and commit it as a single commit and open a new PR or force push over your existing branch.  With the merge conflicts I can't merge it.

**Comment by neopilou** [comment]
Close this PR and open new [293](https://github.com/home-assistant/home-assistant-android/pull/293)

**Pull request closed without merging**